.github/workflows/release-published.yml

```diff
@@ -0,0 +1,22 @@
+name: Checks after release is published
+on:
+  release:
+    types: ['published']
+
+jobs:
+  validate-github-action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: k14s/setup-k14s-action@v1
+        with:
+          only: ytt
+      - run: |
+          ytt version
+          version=`ytt version`
+          release_version="${GITHUB_REF:1}"
```

Comment:
Is github_ref the best way to get the release version? I'm not sure what gets set in there, it seems dependent on the type of trigger.

I see some StackOverflow answers suggesting
`${{ github.event.release.tag_name }}`